build.sbt

```diff
@@ -63,6 +62,7 @@ val sharedSettings = Seq(
   crossScalaVersions := Seq(scalaVersion.value, "2.12.14"),
   javacOptions ++= Seq("-source", "1.8", "-target", "1.8"),
   doc / javacOptions := Seq("-source", "1.8"),
+  versionScheme := Some("early-semver"),
```

Comment:
sbt throws some annothing warnings without this I don't think it does anything functional though